packages/contract-artifacts/CHANGELOG.json

```diff
@@ -10,6 +10,10 @@
                 "note": "Prune fields from EthBalanceChecker",
                 "pr": 1876
             },
+            {
+                "note": "Added solc's devdoc output to ERC20Token artifact",
+                "pr": 1878
+            },
```

Comment:
Wasn't devdoc added to all artifacts in @xianny's PR? If so, maybe we can remove this CHANGELOG entry?

Reply:
https://github.com/0xProject/0x-monorepo/pull/1878/commits/95e02b8f11e15d23dfe562e8e4415dcb6320506d